### PR TITLE
Throw better error from com.ibm.lang.management.ProcessorUsage

### PR DESCRIPTION
**Pull request by pshipton**
Issue #8709
Issue #9506

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile osx jdk11

File: jcl/src/jdk.management/share/classes/com/ibm/lang/management/ProcessorUsage.java [hunk 1]
```diff
@@ -80,17 +80,32 @@ public ProcessorUsage() {
...
-		   (idle < -1) ||
-		   (wait < -1) ||
-		   (busy < -1) ||
-		   (id < -1) ||
-		   (online < 0) || (online > 1) ||
-		   (timestamp < 0) ||
-		   ((user >= 0) && (system >= 0) && (wait >= 0) && (busy >= 0) && (busy < (user + system + wait))))
-		{
-			throw new IllegalArgumentException();
+		if (user < -1) {
```
Comment (DanHeidinga) [inline jcl/src/jdk.management/share/classes/com/ibm/lang/management/ProcessorUsage.java:83]: Does it make sense to also include the "bad" value in the exception message?

Comment (pshipton) [reply jcl/src/jdk.management/share/classes/com/ibm/lang/management/ProcessorUsage.java:83]: Excellent idea

**Comment by pshipton** [comment]
jenkins compile osx jdk11

**Comment by pshipton** [comment]
Running a grinder to see if the original failures can be reproduced.
https://ci.eclipse.org/openj9/view/Test/job/Grinder/894

**Comment by pshipton** [comment]
Current build is https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_mac_Personal/515

**Comment by DanHeidinga** [comment]
jenkins test osx jdk11

**Comment by pshipton** [comment]
jenkins test sanity osx jdk11

**Pull request merged**